### PR TITLE
Feature/31/user can see team members details

### PR DESCRIPTION
**Pull request by LornaHarwood**
**Changes made:**

- Added new columns to team_members table to have role, thumbnail and billable.
- Updated the projects.rake file to pull this data from 10k ft
- Updated the index view page to display each team member with their role and thumbnail picture
- It now lists 100 users per page instead of 20 as the default set

### CODE REVIEW

**Comment by LornaHarwood** [comment]
[User can see each team member's details](https://trello.com/c/IjeDHfQ6/31-user-can-see-each-team-members-details)

**Comment by LornaHarwood** [comment]
[User can see each team member's details](https://trello.com/c/IjeDHfQ6/31-user-can-see-each-team-members-details)

File: app/views/projects/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<h1>Projects</h1>
 <% @projects.each do |project| %>
   <h2><%= project.name %></h2>
     <% project.team_members.each do |team_member| %>
       <ul>
+        <%= image_tag team_member.thumbnail %>
         <li><%= team_member.name %></li>
+        <li><%= team_member.role %></li>
```
Comment (erbridge) [inline app/views/projects/index.html.erb]: What happens if either of these are missing from the fetched data? There are no validations on the data, so it's currently allowed for `role` and `thumbnail` to be `nil`.

Comment (LornaHarwood) [reply app/views/projects/index.html.erb]: At the moment the return of nil is fine. It just means it doesn't display anything on the views page.
We will ensure the validations are added in a later branch once we have the basic first pass for the dashboard

Comment (erbridge) [reply app/views/projects/index.html.erb]: Fine :)

**Pull request merged**